opm/output/eclipse/EclipseIO.hpp

```diff
@@ -28,6 +28,7 @@
 #include <memory>
 #include <string>
 #include <vector>
+#include <optional>
```

Comment:
it is well known most things are different in snåsa, but the alphabet should be ordered the same ;p